src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  justify-content: center;
+  list-style-type: none;
+}
+
+.nav__item {
+  position: relative;
+  text-transform: uppercase;
+  margin-left: 20px;
+}
+
+.nav__item:first-of-type {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: inherit;
+  text-decoration: none;
+}
+
+a.is-active {
+  color: #00acdc;
+}
+
+.nav__item a:hover {
+  cursor: pointer;
+  text-decoration: underline solid #00acdc 40%;
+  color: #00acdc;
+}
+
+a.is-active::after {
```

Comment:
better to write .nav_link.is-active, or just .is-active, it's good practice to use only classes for styling elements